### PR TITLE
Added dependencies in readme

### PR DESCRIPTION
**Pull request by flip111**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -176,12 +176,17 @@ Instead we need to modify layout A's template, and mark the ``description`` anno
 Running Portia
 --------------
 
-Checkout the repository:
+### Dependencies ###
```
Comment (ruairif) [inline README.md]: This should be a sub section of `Running Portia Locally`.

File: README.md [hunk 2]
```diff
@@ -176,12 +176,17 @@ Instead we need to modify layout A's template, and mark the ``description`` anno
...
+You might need to install the following packages
```
Comment (ruairif) [inline README.md]: Mention this is for Ubuntu or Debian. OSX and Windows users can install the packages as binaries from pip.

"When running Portia on Ubuntu or Debian systems you may need to install the following dependencies:"

File: README.md [hunk 3]
```diff
@@ -176,12 +176,17 @@ Instead we need to modify layout A's template, and mark the ``description`` anno
...
 
-    git clone https://github.com/scrapinghub/portia
+    sudo apt-get install libxml2-dev libxslt-dev python-dev zlib1g-dev libffi-dev libssl-dev
```
Comment (ruairif) [inline README.md]: I think build-essential should be here too.

Are you sure about the zlib dependency? Is so do you know which library requires it?

Comment (flip111) [inline README.md]: build-essential was already installed on my system so i didn't add it.

zlib was needed i think by numpy. If you really want to know i will start from scratch and show you the log file. Let me know if this is needed.

**Pull request merged**